docs/source/upcoming_release_notes/1205-Add_MSTA_field_and_homed_status_to_EpicsMotorInterface.rst

```diff
@@ -0,0 +1,32 @@
+1205 Add MSTA field and homed status to EpicsMotorInterface
+#################
+
+API Breaks
+----------
+- N/A
+
+Features
+--------
+- MstaEnum: Enum describing the motor record MSTA bits
+
+Device Updates
+--------------
+- EpicsMotorInterface: Add a "raw" MSTA value, as well as the interpreted
+                       values as a dictionary. Adds a "homed" property based
+                       based on this.
```

Comment:
I think the only thing missing is including that you also did this separately for IMS and Newport